lib/wallet/src/Cardano/Wallet/DB/Pure/Implementation.hs

```diff
@@ -132,6 +133,8 @@ import GHC.Generics
 
 import qualified Data.Map.Strict as Map
 
+import qualified Debug.Trace as TR
```

Comment:
see below

Reply:
it was the case for some commit, but not in the latests one